src/main/java/org/jitsi/videobridge/xmpp/ComponentImpl.java

```diff
@@ -477,4 +478,15 @@ public void stop(BundleContext bundleContext)
             common.stop(bundleContext);
         }
     }
+
+    /**
+     * Returns the {@link Videobridge} instance that is managing conferences
+     * for this component. Returns <tt>null</tt> if no instance is running.
+     *
+     * @return the videobridge instance, <tt>null</tt> when none is running.
+     */
+    public Videobridge getVideobridge()
```

Comment:
I haven't checked if the methods are ordered alphabetically in this class, but if they are then it would be good to put this one in the order as well.

Reply:
They are indeed. Fixed. Please squash :)